src/widgets/roller/lv_roller.h

```diff
@@ -33,13 +33,17 @@ extern "C" {
  **********************/
 
 /** Roller mode.*/
-enum _lv_roller_mode_t {
+#ifdef DOXYGEN
+enum lv_roller_mode_t {
+#else
+enum {
+#endif /*DOXYGEN*/
     LV_ROLLER_MODE_NORMAL, /**< Normal mode (roller ends at the end of the options).*/
     LV_ROLLER_MODE_INFINITE, /**< Infinite mode (roller can be scrolled forever).*/
 };
 
 #ifdef DOXYGEN
-typedef _lv_roller_mode_t lv_roller_mode_t;
+typedef enum lv_roller_mode_t lv_roller_mode_t;
 #else
 typedef uint8_t lv_roller_mode_t;
```

Comment:
We `typedef` some enums to `uint8_t` to allow defining them as a bitfields (e.g. ` lv_roller_mode_t mode : 1;`)

However this practice is used inconsistently in LVGL now and I'm not sure if it's worth the loss in readability. 
Could you update these like this?
```c
typedef enum {
    LV_ROLLER_MODE_NORMAL, /**< Normal mode (roller ends at the end of the options).*/
    LV_ROLLER_MODE_INFINITE, /**< Infinite mode (roller can be scrolled forever).*/
}lv_roller_mode_t;

...

//In lv_roller_t
uint32_t mode : 1;
```